runtime/lib_ksils12.c

```diff
@@ -919,29 +1009,65 @@ rsksifileDestruct(ksifile ksi) {
 
 	ctx = ksi->ctx;
 
+	/* Deregister ksifile so it is not used by signer thread anymore. Note that files are not closed yet! */
+	rsksiDeregisterKsiFile(ctx, ksi);
```

Comment:
no return error verification

Reply:
This should be fixed during another task where overall error handling is improved (and fixed).